L1Trigger/L1TTrackMatch/python/L1TkHTMissEmulatorProducer_cfi.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
+L1TrackerEmuHTMiss = cms.EDProducer("L1TkHTMissEmulatorProducer",
+    L1TkJetEmulationInputTag = cms.InputTag("L1TrackJetsEmulation", "L1TrackJets"),
+    L1MHTCollectionName = cms.string("L1TrackerEmuHTMiss"),
+    jet_maxEta = cms.double(2.4),
+    jet_minPt = cms.double(5.0),
+    jet_minNtracksLowPt=cms.int32(2),
```

Comment:
Make the spacing around the equal signs (` = `) uniform.

Reply:
fixed